commands/checklist.go

```diff
@@ -30,31 +30,31 @@ func (command *ChecklistCommand) Execute([]string) error {
 		return err
 	}
 
-	printCheckfile(pipelineName, config, target.Client().URL())
+	printCheckfile(pipelineName, config, target.Client().URL(), target.Team().Name())
 
 	return nil
 }
 
-func printCheckfile(pipelineName string, config atc.Config, url string) {
+func printCheckfile(pipelineName string, config atc.Config, url, teamName string) {
 	orphanHeaderName := "misc"
 	if len(config.Groups) == 0 {
 		orphanHeaderName = pipelineName
 	}
 
 	for _, group := range config.Groups {
-		printGroup(pipelineName, group, url)
+		printGroup(pipelineName, group, url, teamName)
 	}
 
 	miscJobs := orphanedJobs(config)
 	if len(miscJobs) > 0 {
-		printGroup(pipelineName, atc.GroupConfig{Name: orphanHeaderName, Jobs: miscJobs}, url)
+		printGroup(pipelineName, atc.GroupConfig{Name: orphanHeaderName, Jobs: miscJobs}, url, teamName)
 	}
 }
 
-func printGroup(pipelineName string, group atc.GroupConfig, url string) {
+func printGroup(pipelineName string, group atc.GroupConfig, url, teamName string) {
 	fmt.Printf("#- %s\n", group.Name)
 	for _, job := range group.Jobs {
-		fmt.Printf("%s: concourse.check %s %s %s\n", job, url, pipelineName, job)
+		fmt.Printf("%s: concourse.check %s %s %s %s\n", job, url, pipelineName, job, teamName)
```

Comment:
Putting team last seems a little odd since the hierarchy is team > pipeline > job